webhooks-extension/src/components/WebhookCreate/WebhookCreate.js

```diff
@@ -84,6 +83,14 @@ class WebhookCreatePage extends Component {
         showNotification: true
       });
     }
+    if(this.props.getServiceAccountsErrorMessage){
+      this.setState({
+        notificationMessage: this.props.getServiceAccountsErrorMessage,
```

Comment:
should this be 
```diff
-notificationMessage: this.props.getServiceAccountsErrorMessage,
+notificationMessage: this.props.getServiceAccountsErrorMessage(),
```

?

Reply:
i see the naming is confusing. ill fix this.